mirror-maker-2/active-passive-ssl/config/connect-mirror-maker-template.properties

```diff
@@ -0,0 +1,93 @@
+# Check the source here: https://github.com/apache/kafka/blob/trunk/connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorMakerConfig.java
+
+# Kafka datacenters
+clusters={{.Env.SOURCE_NAME}}, {{.Env.TARGET_NAME}}
+{{.Env.SOURCE_NAME}}.bootstrap.servers={{.Env.SOURCE_BOOTSTRAP_SERVERS}}
+{{.Env.TARGET_NAME}}.bootstrap.servers={{.Env.TARGET_BOOTSTRAP_SERVERS}}
+
+# This configuration is for active -> passive mirroring
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.enabled=true
+{{.Env.TARGET_NAME}}->{{.Env.SOURCE_NAME}}.enabled=false
+
+topics={{.Env.TOPICS}}
+groups={{.Env.GROUPS}}
+
+# Keep topic names (do not append cluster prefix)
+{{- if eq .Env.TARGET_TOPICS_ADD_CLUSTER_PREFIX "true" }}
+replication.policy.class=org.apache.kafka.connect.mirror.DefaultReplicationPolicy
+{{- else}}
+replication.policy.class=org.apache.kafka.connect.mirror.IdentityReplicationPolicy
+{{- end}}
+
+topics.exclude=.*[\-\.]internal, .*\.replica, __consumer_offsets
+groups.exclude=console-consumer-.*, connect-.*, __.*
+
+#target and source configurations. Default value for the following settings is 3.
+#If you want more details about those internal configurations, please see https://docs.confluent.io/home/connect/userguide.html#kconnect-internal-topics
+#and https://docs.confluent.io/platform/current/connect/references/allconfigs.html#distributed-worker-configuration
+{{.Env.TARGET_NAME}}.config.storage.replication.factor=3
+{{.Env.SOURCE_NAME}}.config.storage.replication.factor=3
+#
+{{.Env.TARGET_NAME}}.offset.storage.replication.factor=3
+{{.Env.SOURCE_NAME}}.offset.storage.replication.factor=3
+#
+{{.Env.TARGET_NAME}}.status.storage.replication.factor=3
+{{.Env.SOURCE_NAME}}.status.storage.replication.factor=3
+
+# MirrorMaker configuration. Default value for the following settings is 3
+offset-syncs.topic.replication.factor=3
+heartbeats.topic.replication.factor=3
+checkpoints.topic.replication.factor=3
+
+tasks.max=10
+replication.factor=3
+refresh.topics.enabled=true
+refresh.topics.interval.seconds=30
+sync.group.offsets.interval.seconds=10
+emit.checkpoints.interval.seconds=10
+
+sync.topic.configs.enabled={{.Env.SYNC_TOPIC_CONFIGS}}
+sync.topic.acls.enabled={{.Env.SYNC_TOPIC_ACLS}}
+
+# Enable heartbeats and checkpoints
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.emit.checkpoints.enabled=true
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.sync.group.offsets.enabled = true
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.emit.heartbeats.enabled=true
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.sync.group.offsets.interval.seconds=10
+
+{{.Env.TARGET_NAME}}->{{.Env.SOURCE_NAME}}.emit.heartbeats.enabled=false
+{{.Env.TARGET_NAME}}->{{.Env.SOURCE_NAME}}.emit.checkpoints.enabled=false
+{{.Env.TARGET_NAME}}->{{.Env.SOURCE_NAME}}.sync.group.offsets.enabled = false
+
+# allow very large sizes for messages, as default is 1MB
+{{.Env.SOURCE_NAME}}.consumer.max.request.size=1572864000
```

Comment:
where did these numbers come from?

Reply:
The thought was to allow a large value (1.5 G) so that all messages are included, but it won't cause OOM either

Reply:
> The thought was to allow a large value (1.5 G)

Pedanticism ahead: I think I was confused because the number isn't exactly 1.5 Gigabytes (base 10) or 1.5 Gibibytes (base 2) so I thought it might be something special. Upon inspection it looks to be 1000 * 1.5 mebibyte (i.e. `1.5 * 1024 * 1024 * 1000` and not `1.5 * 1024 * 1024 * 1024` or `1.5 * 1000 * 1000 * 1000`). None of this warrants a change.

Reply:
Right.. I just took 150 MB and added 000 at the end :D . I'll reduce to 1GB for clarity